bitnami/elasticsearch/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 19.2.2
+version: 19.2.3
```

Comment:
Since this is adding a new feature, could you increase the minor instead? I.e. to `19.3.0`.